app/models/case_log.rb

```diff
@@ -51,6 +51,10 @@ class CaseLog < ApplicationRecord
                            end
                          }
 
+  scope :search_by_id, ->(id) { where(id: id) }
+  scope :search_by_tenancy_code, ->(code) { where(tenancy_code: code) }
```

Comment:
I was thinking a good naming convention would be `filter_by_x` for something that expects to match exactly like these, and `search_by` for something matching less exactly i.e. `LIKE` or some other fuzzy match.

Reply:
Good spot, I went for the opposite - will amend!